labs/data_encoding_health.py

```diff
@@ -0,0 +1,116 @@
+
+import pandas as pd
+from helpers.dslabs_functions import get_variable_types, encode_cyclic_variables, dummify
+import numpy as np
+
+
+def calulate_mean_age(age_categories):
+    mean_ages = []
+    for value in age_categories:
+        split = value.split()
+        age1, age2 = int(split[1]), split[3]
+        if age2.isdigit():
+            mean_ages.append((age1+int(age2))/2)
+        else:
+            mean_ages.append(85)
```

Comment:
Where is this number from?

Reply:
85 is very random. Don't know if we want to change that. But my reasoning is that it is a reasonable number as I would guess it is close to the mean